build/build-static-binaries.sh

```diff
@@ -9,7 +9,13 @@ tags=${2-}
 source "$(dirname "${0}")"/build-common.sh
 
 time make STATIC=1 build
+# Build test binaries. Note that the acceptance binary will not be included in
+# the archive, but is instead uploaded directly by push-aws.sh.
+#
+# TODO(tamird,jordan): consider running race-enabled acceptance tests to
+# expose data races in the acceptance tests themselves.
 time make STATIC=1 testbuild PKG=./... TAGS="$tags acceptance"
+time make STATIC=1 testbuild PKG=./... TAGS="$tags acceptance" GOFLAGS=-race TESTBUILDPREFIX=race.
```

Comment:
doubles the build time, no? I'm worried that once I add `deadlock` in here we're going to wait **forever** for the `master` binaries to bake. How do you feel about doing this in a nightly instead (with the baking succeeding a requirement for running the nightlies that need these binaries).